hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/DatanodeUsageInfo.java

```diff
@@ -180,11 +180,13 @@ public int hashCode() {
    *
    * @return Protobuf HddsProtos.DatanodeUsageInfo
    */
-  public DatanodeUsageInfoProto toProto(int clientVersion) {
-    return toProtoBuilder(clientVersion).build();
+  public DatanodeUsageInfoProto toProto(int clientVersion,
+                                        long containerCount) {
+    return toProtoBuilder(clientVersion, containerCount).build();
   }
 
-  private DatanodeUsageInfoProto.Builder toProtoBuilder(int clientVersion) {
+  private DatanodeUsageInfoProto.Builder toProtoBuilder(int clientVersion,
+                                                        long containerCount) {
```

Comment:
Instead of adding `containerCount` as a parameter to these methods, please add it to the `DatanodeUsageInfo` class.